CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+**0.4.0**
+1. Memory pool improvements: upper bound size limit introduced (default 300Mb) altogether with cleanup strategy (the lowest fee rate transaction removed first).
```

Comment:
we have also memortyPool.minFeeRate  (default 0 ) - to set optionally a minimum fee rate for a tx to be accepted in mempool

Reply:
thanks, added this info